src/components/editor/Tabs.tsx

```diff
@@ -46,25 +62,29 @@ export class Tabs extends React.Component<{
     e.preventDefault();
   }
 
-  onDoubleClick = (e: MouseEvent<any>) => {
-    return this.props.onDoubleClick && this.props.onDoubleClick();
-  }
+  onDoubleClick   = (e: MouseEvent<any>)  => { this.props.onDoubleClick(); };
```

Comment:
`onDoubleClick` is optional, so it may be undefined, hence the reason we used the `... && ...` syntax.

Reply:
I create [**defaultProps** call stubs](https://github.com/MaxGraey/WebAssemblyStudio/blob/980998b0ddb66b873aaab14042901cc7d2343b1c/src/components/editor/Tabs.tsx#L106) for this. In my opinion it make code much clearer.

Reply:
Ah, I see.